Lib/test/test_tarfile.py

```diff
@@ -225,18 +225,17 @@ def test_add_dir_getmember(self):
         self.add_dir_and_getmember('bar')
         self.add_dir_and_getmember('a'*101)
 
-    @unittest.skipIf(
```

Comment:
Please skip the test if hasattr(os, 'getuid') is false or hasattr(os, 'getgid') is false.